shared/loadgenerator/loadgenerator.go

```diff
@@ -208,5 +225,5 @@ func (gr *GeneratorResults) ErrorsPercentage(idx int) float64 {
 			errors = errors + count
 		}
 	}
-	return float64(errors*100) / float64(errors+successes)
+	return successes, errors
```

Comment:
we can return errs and total? So, that we dont have to do an additional calc of total = succ + err above

Reply:
Good idea. Done.